pyston/data_processor.py

```diff
@@ -51,13 +51,10 @@ def get_processors(self, resource_class):
 
 
 class DataProcessor(object):
-    def __init__(self, resource, form, inst, via):
+    def __init__(self, resource, form, *args, **kwargs):
```

Comment:
chci dovoli posílat do konstruktoru cokoli, jelikož to vyžadují modeldataprocesory